add metaval config (#678)
nf-core/taxprofiler: docs/usage.md

Note that the multi-sample tables from the 'native' tools in each folders are [not inter-operable](https://taxpasta.readthedocs.io/en/latest/tutorials/getting-started/) with each other as they can have different formats and can contain additional and different data. In this case we refer you to use the standardised and merged output from Taxpasta, as described above.

### Generate inputs for downstream pipeline

#### genomic-medicine-sweden/metaval

[genomic-medicine-sweden/metaval](https://github.com/genomic-medicine-sweden/metaval) is a Nextflow pipeline for validation of taxonomic classifications generated by `nf-core/taxprofiler`. It assesses whether identified taxa are true or false discoveries by analyzing assigned reads using sequence alignment (BLAST) and reference-based read mapping. At the moment, `genomic-medicine-sweden/metaval` supports taxa classified by `Kraken2`, `Centrifuge` and `DIAMOND` for both Illumina and Nanopore sequencing data.

`metaval` needs minimum following files:

- FASTAQ reads (raw, filtered, or after host-removal)
- One taxonomic classification table
- one taxpasta table

> To generate the required inputs for `genomic-medicine-sweden/metaval`, include the metaval profile using `-profile <your_other_configs>,metaval`, and don't modify `--run_profile_standardisation` or `--taxpasta_add_lineage`.

Reviewer: ```suggestion
To generate the required inputs for `genomic-medicine-sweden/metaval`, include the `metaval` profile using `-profile <your_other_configs>,metaval`.

> [!DANGER]
> Do not modify `--run_profile_standardisation` or `--taxpasta_add_lineage`.
> The output files from these flags are mandatory input for `metaval`
```

### Generate inputs for downstream pipeline

#### genomic-medicine-sweden/metaval

[genomic-medicine-sweden/metaval](https://github.com/genomic-medicine-sweden/metaval) is a Nextflow pipeline for validation of taxonomic classifications generated by `nf-core/taxprofiler`. It assesses whether identified taxa are true or false discoveries by analyzing assigned reads using sequence alignment (BLAST) and reference-based read mapping. At the moment, `genomic-medicine-sweden/metaval` supports taxa classified by `Kraken2`, `Centrifuge` and `DIAMOND` for both Illumina and Nanopore sequencing data.

`metaval` needs minimum following files:

- FASTQ reads (raw, filtered, or after host-removal)
- One taxonomic classification table
- One taxpasta table

To generate the required inputs for `genomic-medicine-sweden/metaval`, include the `metaval` profile using `-profile <your_other_configs>,metaval`.

> [!DANGER]
> Do not modify `--run_profile_standardisation` or `--taxpasta_add_lineage`.
> The output files from these flags are mandatory input for `metaval`

We generally recommend cleaning up and preprocessing your reads prior passing to `metaval`.

> If you want to filter short FASTQ input files reads to `metaval` (as in, your reads are not already preprocessed), include `--perform_shortread_qc --perform_shortread_complexityfilter --save_complexityfiltered_reads`.

Reviewer: ```suggestion
If you want to filter your short FASTQ input files reads for use in `metaval` (as in, your reads are not already preprocessed), additionally include `--perform_shortread_qc --perform_shortread_complexityfilter --save_complexityfiltered_reads`.
```

If you want to filter long FASTQ input files reads, specify `--perform_longread_qc --save_preprocessed_reads`.

If you want to remove host reads from short-read input to `metaval`, include `--perform_shortread_hostremoval`.
If you want to remove host reads from long-read input to `metaval`, include `--perform_longread_hostremoval --save_hostremoval_unmapped --hostremoval_reference '/path/to/host/genome'`.

> By default, the `metaval` config will turn on all of `--run_kraken2 --kraken2_save_readclassifications`, `--run_centrifuge` or `--run_diamond`.

Reviewer: ```suggestion
By default, the `metaval` config will turn on all of `--run_kraken2 --kraken2_save_readclassifications`, `--run_centrifuge` and `--run_diamond`.
```
